src/components/MultiGesture.js

```diff
@@ -110,6 +110,9 @@ class MultiGesture extends React.Component {
   }
 
   render() {
+    if (this.props.showOverlay) {
+      return <div>{this.props.children}</div>
```

Comment:
Rather than re-render the MultiGesture when `showOverlay` changes (which will re-render all content), let's add an additional condition to the early return in [`handleGestureSegment`, `handleGestureEnd`, and `handleKeyboard`](https://github.com/cybersemics/em/blob/27f3afe29706550e47f54e4f6f05be01613ee0d3/src/shortcuts.js#L96-L171) so that they exit when the toolbar overlay is active.

Reply:
Okay.